testcases/client-server/dnf-FTP/dummy.spec

```diff
@@ -0,0 +1,31 @@
+Summary:        This is a dummy rpm-test 
+Name:           dummy-rpm
+Version:        1.0
+#Release:        %(echo `date '+%%Y%%m%%d.%%H%%M%%S'`)
+Release:	1
+Epoch:          0
+Group:          Development/Interpreters
+License:        GPL+
+BuildArch:      noarch
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root
```

Comment:
Not needed

Reply:
I can't remove the License, since the rpmbuild requires it, however I removed the other lines.

Reply:
I meant BuildRoot: tag only. So BuildArch should be there to keep from building for all arches.